components/occurrences/BuildOccurrenceDetails.js

```diff
@@ -28,12 +28,20 @@ const BuildOccurrenceDetails = ({ occurrence }) => {
       <div className={styles.detailSummary}>
         <div>
           <p className={styles.title}>Build</p>
-          <ExternalLink
-            href={occurrence.sourceUri}
-            label={"View source"}
-            className={styles.rightMargin}
-          />
-          <ExternalLink href={occurrence.logsUri} label={"View logs"} />
+          {occurrence.sourceUri ? (
+            <ExternalLink
```

Comment:
The `ExternalLink` component is already checking if it has a valid link, maybe it could also take a fallback component to render instead of null?

Reply:
Good thinking.